boudicca.base/publisher-event-html-app/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
```

Comment:
why does the publisher have it's own .gitignore?

Reply:
i have no idea :D i removed all unneeded .gitignore files, there where more of them